portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,125 @@
+			<h1>Grant Commodore</h1>
+			<img src="images/profile.jpg" class="profile">
+			<h2>Sophmore Computer Science Major, Mathematics Minor</h2>
+			<h2 style="padding-bottom: 15px;">Aspiring Software Engineer</h2>
+		</div>
+	</div>
+	<section class="aboutMeSection" , id="aboutme">
+		<h2 class="sectionHeader">About Me</h2>
+		<hr class="round">
+		<div class="row">
+			<div id="education" , class="column">
+				<h2>Education</h2>
+				<img class="rowImage" src="images/university.png">
+				<div>
+					<button id= "eduButton",class="button", onclick="showModal()">Learn More</button>
+				</div>
+			</div>
+			<div id="interests" , class="column">
+				<h2>Interests</h2>
+				<img class="rowImage" src="images/idea.png">
+				<div>
+					<button id= "interestsButton",class="button", onclick="showModal()">Learn More</button>
+				</div>
+			</div>
+			<div id="funFacts" , class="column">
+				<h2>Fun Facts</h2>
+				<img class="rowImage" src="images/football-ball.png">
+				<div>
+					<button id= "funFactsButton",class="button", onclick="showModal()">Learn More</button>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="sectionRow" , id="projects">
+		<h2 class="sectionHeader">Projects</h2>
+		<hr class="round">
+		<div class="row">
+			<div class="column">
+				<h2>Web</h2>
+				<img class="rowImage" src="images/laptop.png">
+				<div>
+					<button id= "webButton", class="button", onclick="showModal()">See Examples</button>
+				</div>
+			</div>
+			<div class="column">
+				<h2>Mobile</h2>
+				<img class="rowImage" src="images/mobile.png">
+				<div>
+					<button id= "mobileButton", class="button", onclick="showModal()">See Examples</button>
+				</div>
+			</div>
+			<div class="column">
+				<h2>Github</h2>
+				<img class="rowImage" src="images/usb.png">
+				<div>
+					<button id= "githubButton", class="button", onclick="showModal()">See Examples</button>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="sectionRow" , id="skills">
+		<h2 class="sectionHeader">Skills</h2>
+		<hr class="round">
+		<div class="row">
+			<div class="column">
+				<h2>List of Skills</h2>
+				<img class="rowImage" src="images/desktop.png">
+				<div>
+					<button id= "skillsButton", class="button", onclick="showModal()">Show Details</button>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<button onclick="topFunction()" id="myBtn" title="Go to top">Back to Top</button>
+
+	<div id="content">
+		<h2 class="sectionHeader" , style="padding-top:10px">Connect with Me</h2>
+		<p style="text-align:center; font-size:19px;">
+			Thank you for taking to time to learn a little bit about me!
+			If you have any questions, please feel free to connect on LinkedIn.
+		</p>
+		<button class="button", onclick="window.location.href='https://www.linkedin.com/in/grant-commodore-753b98171';">Let's Connect!</button>
+		<div id="fact-container"></div>
+		<div style="text-align:left">Icons made by <a href="https://www.flaticon.com/authors/freepik"
```

Comment:
This looks kind of funny (inlined <a>) but I'm not sure what you should do to clean this up. @jalexanderqed any suggestions?